src/Config.js

```diff
@@ -25,7 +25,9 @@ var Config = function (root, configpath) {
     this.pluginsManager = new PluginsManager(this);  // Do we need back ref?
     this.emit('loaded');
     this.on('server:init', this.attachRoutes.bind(this));
-    this.parsed_opts = {};  // Default. TODO better option management.
+    this.parsed_opts = {
+        renderer: 'carto'
```

Comment:
Without it tests were failing. Somewhere there is a difference between project loading for tests and for real.